OpenUpSA/wazimap-ng-ui
# removing the anchor headline links on the rich data view

## Description
Hiding the category anchor headline links on rich data view

## Related Issue
https://github.com/OpenUpSA/wazimap-ng-ui/issues/148

## How to test it locally
* Run the project
* Confirm that the categories on rich data panel don't have the headline links

## Screenshots
Before : 
![image](https://user-images.githubusercontent.com/53019884/100353336-6a35aa80-2fff-11eb-9d7c-8335f7a51762.png)

After : 
![image](https://user-images.githubusercontent.com/53019884/100353408-80436b00-2fff-11eb-8d31-938f5c22faef.png)


## Changelog

### Added

### Updated
* modified `addCategory()` function in `category.js` to add `hidden` class to the link items(`.category-header__link`)
 
### Removed


## Checklist


### Pull Request


### Commits


### Code Quality


### Testing
